packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -359,6 +385,55 @@ export class NodeProxyTarget extends ProxyTarget<Anchor> {
             .map((fieldKey) => this.proxifyField(fieldKey, false))
             .values();
     }
+
+    public createField(fieldKey: FieldKey, newContent: ITreeCursor): EditableField | undefined {
+        assert(!this.has(fieldKey), "The field already exists.");
+        const fieldKind = this.lookupFieldKind(fieldKey);
+        assert(
```

Comment:
It would be better to include this checking in the switch statement, so we are only handling the different multiplicities in one place.

Reply:
done